doc/check.texi

```diff
@@ -993,6 +993,17 @@ repetitive code that is hard to read.  For your convenience Check
 provides a set of functions (actually macros) for testing often used
 conditions.
 
+@findex check_set_max_msg_size
```

Comment:
I think this could be moved to the end of the section, as it would not be expected to be called as frequently as the others. Or, do you disagree?

Reply:
I think that placing it at the end would make it "disappear" for someone who would say "ah, a list of all kind of assertions, I'll skip to the next section and come back here if I want to locate a specific assertion function". The other option is to make it an explicit section of its own but that seems an overkill for a single simple function.